### ISSUE TITLE
Unicode should be formated in UTF-8

### ISSUE THREAD

**Issue by willcbaker** [baalexander/rospy_message_converter#11]
when converting to ros string, should be decoded as utf-8, as unicode is not supported as specified by http://wiki.ros.org/msg

> unicode strings are currently not supported as a ROS data type. utf-8 should be used to be compatible with ROS string serialization. In python 2, this encoding is automatic for unicode objects, but decoding must be done manually. In python 3, both encoding and decoding are automatic.

Proposed fix:
_convert_to_ros_primitive function in 
src/rospy_message_converter/message_converter.py
line 129:

```
def _convert_to_ros_primitive(field_type, field_value):
    if field_type == 'string':
        field_value = field_value.encode('utf-8')
    return field_value
```

**Comment by baalexander** [comment]
@willcbaker thanks for bringing this to my attention. Your proposed fix looks good. Would you mind submitting a PR with a comment above the conditional linking to the ros.org/msg explanation? A major nice to have would also be a test case that fails in python2 before this fix was applied.

**Comment by willcbaker** [comment]
@baalexander this good to go?

**Comment by Jaime5** [comment]
this is a pretty important fix, will this ever be merged?

**Comment by mintar** [comment]
Fixed in #13.
